### PR TITLE
Issue 210: add hsamag bags for 3ifb,ctrg,syndikat

### PR DESCRIPTION
**Pull request by shadow-fa**
see issue #210.

### CODE REVIEW

File: f/assignGear/f_assignGear_3IFB.sqf [hunk 1]
```diff
@@ -182,6 +182,7 @@ _baghatg = "B_AssaultPack_rgr";				// Used by Heavy AT Gunner
 _bagmtrg = "B_Mortar_01_weapon_F";			// used by Mortar gunner
 _bagmtrag = "B_Mortar_01_support_F";		// used by Mortar assistant gunner
 _baghsamg = "B_AA_01_weapon_F";				// used by Heavy SAM gunner
+_baghsamag = "B_HMG_01_support_F";			// used by Heavy SAM assistant gunner **
```
Comment (shadow-fa) [inline f/assignGear/f_assignGear_3IFB.sqf]: how

File: f/assignGear/f_assignGear_ctrg.sqf [hunk 2]
```diff
@@ -183,6 +183,7 @@ _baghatag = "B_HMG_01_support_F";			// used by Heavy AT assistant gunner
 _bagmtrg = "B_Mortar_01_weapon_F";			// used by Mortar gunner
 _bagmtrag = "B_Mortar_01_support_F";		// used by Mortar assistant gunner
 _baghsamg = "B_AA_01_weapon_F";			// used by Heavy SAM gunner
+_baghsamag = "B_HMG_01_support_F";			// used by Heavy SAM assistant gunner **
```
Comment (shadow-fa) [inline f/assignGear/f_assignGear_ctrg.sqf]: did we

File: f/assignGear/f_assignGear_syndikat.sqf [hunk 3]
```diff
@@ -182,6 +182,7 @@ _baghatg = "B_AssaultPack_rgr";				// Used by Heavy AT Gunner
 _bagmtrg = "B_Mortar_01_weapon_F";			// used by Mortar gunner
 _bagmtrag = "B_Mortar_01_support_F";		// used by Mortar assistant gunner
 _baghsamg = "B_AA_01_weapon_F";				// used by Heavy SAM gunner
+_baghsamag = "B_HMG_01_support_F";			// used by Heavy SAM assistant gunner **
```
Comment (shadow-fa) [inline f/assignGear/f_assignGear_syndikat.sqf]: miss this?

File: f/assignGear/f_assignGear_3IFB.sqf (hunk 1 above)
Comment (SamLex) [reply f/assignGear/f_assignGear_3IFB.sqf]: 'cause

File: f/assignGear/f_assignGear_ctrg.sqf (hunk 2 above)
Comment (SamLex) [reply f/assignGear/f_assignGear_ctrg.sqf]: assignGear is the

File: f/assignGear/f_assignGear_syndikat.sqf (hunk 3 above)
Comment (SamLex) [reply f/assignGear/f_assignGear_syndikat.sqf]: worst!

**Pull request merged**